### ISSUE TITLE
Search zone for record

### ISSUE THREAD

**Issue by Kofl** [TechnitiumSoftware/DnsServer#948]
Hi,

either I completely missed it, but it would be super helpful to search for records in a zone (name of entry or data of entry).
So, you would be faster to find and change it.

Thanks

**Comment by Kofl** [comment]
Great, maybe its possible to add batch processing to the filter/search. E.g. ticking "Select All" and then set the TTL for them.

**Comment by ShreyasZare** [comment]
Yes, will try to add similar features to make it a bit easier to manage zones.

**Comment by CRASH-Tech** [comment]
Any news? It's now usable without search domains :(

**Comment by necroave** [comment]
![image](https://github.com/user-attachments/assets/142d8e0d-9c25-49c8-9c69-8fa9bcc83675)
I’m going to be blunt here—this current search system is incredibly frustrating and is slowing everyone down. It’s unacceptable to waste time trying to find basic information. Implementing a search by zones should’ve been done ages ago, and it’s time to get it done now. The user experience is suffering, and it’s affecting productivity. I expect this to be addressed and improved without further delay. Thanks!

**Comment by ShreyasZare** [comment]
Thanks for asking. This feature will take a while since full Clustering feature is being prioritized at the moment. I will try to get this added in the subsequent updates after the next major release.

**Comment by qingxiaoxinrou** [comment]
您好 请问可否将这个功能提前实现 真的太影响效率 感谢！

**Comment by ShreyasZare** [comment]
> 您好 请问可否将这个功能提前实现 真的太影响效率 感谢！

@qingxiaoxinrou 谢谢您的提问。请参阅上面的回复。

**Comment by CRASH-Tech** [comment]
Any news?

**Comment by ShreyasZare** [comment]
> Any news?

@CRASH-Tech The clustering feature is being prioritized for now along with fixing bugs in current release. This will be taken up after clustering is available.

**Comment by alisonarantes** [comment]
Any news?

**Comment by ShreyasZare** [comment]
@alisonarantes Thanks for asking. This will be implemented only after clustering feature is released. Right now, clustering implementation has started and it may take a couple of months to get the next major release. This will be then added in subsequent minor releases.

**Comment by CRASH-Tech** [comment]
Clustering is very, very specific feature, because DNS by design have reservation machanism. But RR search is feature that needs right now. I, and many others can't use it without this basic and simple feature!

**Comment by ShreyasZare** [comment]
> Is it possible to implement at least the records per page within the zone tab to have an option of all (after 500) or increase that number? This way, with an option of all, we can at least do a search through the browser (ctrl + F). BTW, congrats for the amazing application.

@alisonarantes you can do this directly by editing the option html by right clicking the dropdown and clicking Inspect option. Just edit the option that says 500 to a higher number and then use the updated option. Use this workaround till the proper option is made available.

**Comment by ShreyasZare** [comment]
> Clustering is very, very specific feature, because DNS by design have reservation machanism. But RR search is feature that needs right now. I, and many others can't use it without this basic and simple feature!

@CRASH-Tech I agree with this and will try to get it added with the clustering update. The clustering update already needs many UI changes so I can try to add this too with it.

**Comment by ShreyasZare** [comment]
Technitium DNS Server v13.5 is now available that adds filtering option for zone edit view. Do update and let me know your feedback.
